lib/node_modules/@stdlib/array/empty-like/docs/types/index.d.ts

```diff
@@ -437,8 +437,8 @@ declare function emptyLike( x: Complex128Array, dtype?: DataType ): Complex128Ar
 * @example
 * var zeros = require( `@stdlib/array/zeros` );
 *
-* var x = zeros( 2, 'float32' );
-* // returns <Float32Array>[ 0.0, 0.0 ]
+* var x = zeros( 2, 'complex64' );
+* // returns <Complex64Array>[ 0.0, 0.0 ]
```

Comment:
```suggestion
* // returns <Complex64Array>
```

Same comment as above.